packages/backend/src/misc/error.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: MomentQYC and other misskey contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { FastifyReply, FastifyRequest } from 'fastify';
+
+export function ErrorHandling(message: string, reply?: FastifyReply, statusCode?: number): Error {
+	const error = new Error(message);
+	if (process.env.NODE_ENV === 'production') {
+		error.stack = undefined;
+	}
+	if (reply) {
+		reply.code(statusCode ?? 500);
```

Comment:
You can skip this if you do `reply.send(error)`, the send function will pass the error to the default error handler where it'll process the status code.